packages/controls/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "color": "^3.1.3",
     "css-box-model": "^1.2.1",
+    "csstype": "^2.6.5",
```

Comment:
We decided a while ago that it's okay for `@makeswift/controls` to depend on CSS types, see e.g. `css-box-model` dep above.